scopesim/effects/metis_lms_trace_list.py

```diff
@@ -562,13 +569,10 @@ class MetisLMSEfficiency(TERCurve):
     }
 
     def __init__(self, **kwargs):
-        # TODO: Refactor these _class_params?
-        self.meta = copy.copy(self._class_params)
-        assert "grat_spacing" in self.meta, "grat_spacing is missing from self.meta 1"
-        super().__init__(**kwargs)
-        assert "grat_spacing" in self.meta, "grat_spacing is missing from self.meta 2"
+        self.meta = self._class_params
+        self.meta.update(kwargs)
```

Comment:
Removing this `copy()` seems rather dangerous, as now any instance of `MetisLMSEfficiency` can change the values of other instances. E.g.:
```python
from scopesim.effects.metis_lms_trace_list import MetisLMSEfficiency
eff1 = MetisLMSEfficiency(wavelen=4.2, filename="METIS/TRACE_LMS.fits")
print(f"{eff1.meta['eff_max']=}")
eff2 = MetisLMSEfficiency(wavelen=4.2, filename="METIS/TRACE_LMS.fits", eff_max=42)
print(f"{eff2.meta['eff_max']=}")
print(f"{eff1.meta['eff_max']=}")
eff3 = MetisLMSEfficiency(wavelen=4.2, filename="METIS/TRACE_LMS.fits")
print(f"{eff3.meta['eff_max']=}")
```
will print
```
eff1.meta['eff_max']=0.75
eff2.meta['eff_max']=42
eff1.meta['eff_max']=42
eff3.meta['eff_max']=42
```
is that intentional?